templates/zerver/realm_redirect.html

```diff
@@ -44,7 +44,9 @@ <h1 class="get-started">{{ _("Log in to your organization") }}</h1>
         </div>
 
         <div class="bottom-text">
-            {{ _("Need to get your group started on Zulip?") }} <a target="_blank" rel="noopener noreferrer" href="/new/">{{ _("Create a new organization.") }}</a>
+            {% trans %}
+            <a target="_blank" rel="noopener noreferrer" href="/new/">Create a new organization</a> if you don't have one yet.
+            {% endtrans %}
```

Comment:
I think `trans` supports putting the code part into a variable; we use this in `templates/zerver/emails/notify_new_login.html`, for example. Seems valuable to help avoid translators needing to redo their work when we make changes here.

Reply:
Okay, updated!